### ISSUE TITLE
Error 'Setting "TEST_NO_AUTO_START_RUNS" is not set' in server's worker log

### ISSUE THREAD

**Issue by isaacliao** [StanfordBioinformatics/loom#372]
```
[2017-05-05 00:20:24,070: INFO/MainProcess] Received task: api.tasks.process_active_step_runs[49668aa0-5fe0-464e-9a8f-86f3978be263]
[2017-05-05 00:20:24,072: ERROR/MainProcess] Task api.tasks.process_active_step_runs[49668aa0-5fe0-464e-9a8f-86f3978be263] raised unexpected: Exception(u'Setting "TEST_NO_AUTO_START_RUNS" is not set',)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 367, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/celery/app/trace.py", line 622, in __protected_call__
    return self.run(*args, **kwargs)
  File "/Users/iliao/repos/loom/loomengine/master/api/tasks.py", line 80, in process_active_step_runs
  File "/loomengine/loomengine/master/api/__init__.py", line 10, in get_setting
    raise Exception('Setting "%s" is not set' % SETTING)
Exception: Setting "TEST_NO_AUTO_START_RUNS" is not set
```

**Comment by nhammond** [comment]
Isaac, which branch is this? Can you make sure you have merged recent changes from dev? api.tasks is now api.async, and this setting has been removed.

**Comment by isaacliao** [comment]
Okay, I see what happened. There were leftover *.pyc files after I merged. `tasks.py` got deleted but not `tasks.pyc`. Also, they got copied into the Docker image because I was building locally. *.pyc files are excluded by .gitignore, so they never get committed to Git or included in Docker images built remotely from a fresh git clone.

I've deleted all .pyc files and set PYTHONDONTWRITEBYTECODE=True on my development machine so this doesn't happen again.
